sunpy/map/sources/soho.py

```diff
@@ -33,15 +34,31 @@ class EITMap(GenericMap):
     * `SOHO EIT User Guide <https://umbra.nascom.nasa.gov/eit/eit_guide/>`_
 
     """
+    def _handle_deprecated_ctypes(self, ctype1, ctype2):
```

Comment:
This needs to override the `coordinate_system` method on the GenericMap class instead.

Reply:
ok got it